src/storm/api/builder.h

```diff
@@ -72,13 +72,15 @@ std::shared_ptr<storm::models::symbolic::Model<LibraryType, ValueType>> buildSym
 
 template<>
 inline std::shared_ptr<storm::models::symbolic::Model<storm::dd::DdType::CUDD, storm::RationalNumber>> buildSymbolicModel(
-    storm::storage::SymbolicModelDescription const& model, std::vector<std::shared_ptr<storm::logic::Formula const>> const& formulas, bool, bool) {
+    [[maybe_unused]] storm::storage::SymbolicModelDescription const& model,
+    [[maybe_unused]] std::vector<std::shared_ptr<storm::logic::Formula const>> const& formulas, bool, bool) {
```

Comment:
This is now inconsistent, either name all arguments and use maybe unused, or what i think is better here is to not name any of them.